src/main/java/org/veupathdb/service/eda/ss/model/tabular/TabularResponses.java

```diff
@@ -16,21 +19,27 @@ public class TabularResponses {
   private TabularResponses(){}
 
   public enum Type {
-    TABULAR(TABULAR_FORMATTER, MimeTypes.TEXT_TABULAR),
-    JSON(JSON_FORMATTER, MediaType.APPLICATION_JSON);
+    TABULAR(TABULAR_FORMATTER, MimeTypes.TEXT_TABULAR, TABULAR_BINARY_FORMATTER),
+    JSON(JSON_FORMATTER, MediaType.APPLICATION_JSON, TABULAR_BINARY_FORMATTER);
```

Comment:
Need to implement JSON version here.  Also (nit) would prefer the arg order be (formatter, binary-formatter, mime-type), to put those two together.

Reply:
I thought I had this locally, but I must not have committed. Adding back now